python/pyspark/sql/functions.py

```diff
@@ -1648,6 +1648,14 @@ def sort_array(col, asc=True):
 
 # ---------------------------- User Defined Function ----------------------------------
 
+def _wrap_function(sc, func, returnType):
```

Comment:
what's the point of creating a new `_wrap_function` here? To decrease the size of serialized python function?